README.md

```diff
@@ -19,3 +19,7 @@ guideline make sense to you, [open an
 issue](https://github.com/georust/team/issues/new) with your proposal or strike
 up a conversation in [our Discord
 server](https://discord.com/channels/598002550221963289).
+
+## Code of Conduct
```

Comment:
What would you think instead of adding the COC blurb to the existing GUIDELINES file and linking to GUIDELINES here?

Reply:
errr I guess we already link to the guidelines in the paragraph above. 

Since this is a short snippet, I'd be ok with having it here, like you have it, in the guidelines, or both.

Reply:
Good point! I think the guidelines might make more sense for this, since it's less about this repo and more about encouraging the use for other projects